src/app/components/MediaLoader/index.tsx

```diff
@@ -150,16 +152,19 @@ const MediaContainer = ({
     }
   }, [playerConfig, showAds]);
 
+  const playerStyling = (() => {
+    if (orientation === 'portrait') {
+      return styles.mediaContainerPortrait;
+    }
+    if (mediaType === 'liveRadio') {
+      return styles.liveRadioMediaContainer;
+    }
+
+    return styles.mediaContainerLandscape;
+  })();
+
```

Comment:
I don't love this syntax, as I find it hard to read, and it's very subtle knowing that the function has been executed. To improve readability could we do something this?

Either
```
const getPlayerStyling = () => {
  if (orientation === 'portrait') {
    return styles.mediaContainerPortrait;
  }
  if (mediaType === 'liveRadio') {
    return styles.liveRadioMediaContainer;
  }
   return styles.mediaContainerLandscape;
};
```
Then
```
const playerStyling = getPlayerStyling();
```
OR 
```
<div ref={playerElementRef} data-e2e="media-player" css={getPlayerStyling()} />
```

Alternatively, get rid of the function and just get the styling inline:

```
let playerStyling = styles.mediaContainerLandscape;

if (orientation === 'portrait') {
  playerStyling = styles.mediaContainerPortrait;
}
if (mediaType === 'liveRadio') {
  playerStyling = styles.liveRadioMediaContainer;
}
```

Reply:
I prefer using an IIFE here, as it seems wasteful to create a function or a re-assignable variable that's only used once and a little confusing as it implies it's used in multiple places, and to me the anonymous function isn't that large to be unreadable, could use this syntax to make it more understandable that the function is getting invoked?

```ts
const playerStyling = (function stylingDecision() {
    if (orientation === 'portrait') {
      return styles.mediaContainerPortrait;
    }
    if (mediaType === 'liveRadio') {
      return styles.liveRadioMediaContainer;
    }

    return styles.mediaContainerLandscape;
  })();
```

Reply:
I don't mind an IIFE. If we're only using once, makes sense. Guess it's a pattern we don't see too often in back end JS, so we might need to explain to some new devs.

Reply:
@HarveyPeachey I find the original better, so happy to leave it as is. IIFEs are just something I will learn to ~love~ like, once I've got my head around them! Thanks.